.deploy/prod-fss-default.yaml

```diff
@@ -90,7 +90,7 @@ spec:
     - name: MELDEKORTUTBETALINGSGRUNNLAG_V1_URL
       value: https://arena.adeo.no/ail_ws/MeldekortUtbetalingsgrunnlag_v1
     - name: PDL_BASE_URL
-      value: http://pdl-api.default
+      value: http://pdl-api.default/graphql
```

Comment:
same same

Reply:
Ikke særlig fan av å ikke angi avhengigheter eksplisitt.
Men skal ikke insistere dersom det er hva som må til for å få PR gjennom